adobe/react-spectrum
# feat: event coordinates

Closes https://github.com/adobe/react-spectrum/issues/2031

Related https://github.com/adobe/react-spectrum/issues/1998.

Add `clientX` and `clientY` positioning to `onPress` and `onHover` events for implementing ripple or wave effect.

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- Open the storybook for the `Button - Default` component.
- Clicking on different positions of the target element.
- Check the coordinates `x,y` displayed in the Actions panel.
- When triggering the event using the keyboard, the coordinates will be `0,0`.

## 🧢 Your Project:

[Button drip effect - NextUI](https://github.com/nextui-org/nextui/blob/497e06b05e1245f3ad83fc4e2f83a36cc72596c0/packages/components/button/src/use-button.ts#L110-L126).
